frontend/packages/noobaa-storage-plugin/integration-tests/views/createBC.view.ts

```diff
@@ -0,0 +1,169 @@
+import { $, $$, by, element, ExpectedConditions as until, browser } from 'protractor';
+import * as crudView from '@console/internal-integration-tests/views/crud.view';
+import { click } from '@console/shared/src/test-utils/utils';
+import { execSync } from 'child_process';
+import * as _ from 'lodash';
+import { testName } from '@console/internal-integration-tests/protractor.conf';
+
+// Backing Store is the first one in Operator Hub
+export const createBucketClassLink = $('article:nth-child(2) a');
```

Comment:
Can you look for other option? This doesn't look scale-able approach

Reply:
sure, will look into it.

Reply:
https://github.com/openshift/console/pull/4413
I will make changes once 4413 gets merged.